spec/views/posts/show.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe 'posts/show.html.erb', type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
- [ ] Considering you were advised not to use the generate command, so that you can get a hands-on approach to how routing and controllers work. But you used them anyway. Kindly remove this generated file.